tests/test_model_signature.py

```diff
@@ -57,3 +57,14 @@ def test_invalid_identifiers_signature():
     assert _equals(str(signature(model)), '(*, valid_identifier: int = 123, yeah: int = 0) -> None')
     model = create_model('Model', **{'123 invalid identifier!': 123, '!': Field(0, alias='yeah')})
     assert _equals(str(signature(model)), '(*, yeah: int = 0, **data: Any) -> None')
+
+
+def test_kwargs():
```

Comment:
As I said in https://github.com/samuelcolvin/pydantic/pull/1034#issuecomment-592161978, this test would already work without changes in this commit